pyseext/js/PySeExt.GridHelper.js

```diff
@@ -181,5 +181,55 @@ globalThis.PySeExt.GridHelper = {
         }
 
         return columnHeader;
+    },
+
+    /**
+     * Gets the row index with the specified data or index in the grid.
+     * The row is scrolled into view ready for clicking by the caller.
+     *
+     * The grid must be visible.
+     *
+     * @param  {Object|Number} rowData The index of or an object containing the row data for the record to be found.
+     * @return {Object} The data for the row record.
```

Comment:
This needs updating, then I think that's it!
Something like:
`@return {Number} The index for the row record, or undefined if not found`

Reply:
Ah bugger, sorry should have spotted that. Updated